src/urllib3/contrib/_securetransport/bindings.py

```diff
@@ -47,16 +47,6 @@
 from ctypes import CDLL, POINTER, CFUNCTYPE
 
 
-security_path = find_library("Security")
-if not security_path:
-    raise ImportError("The library Security could not be found")
-
-
-core_foundation_path = find_library("CoreFoundation")
-if not core_foundation_path:
-    raise ImportError("The library CoreFoundation could not be found")
-
-
 version = platform.mac_ver()[0]
```

Comment:
Can you please raise an `ImportError` if we're not on macOS at all? This is breaking the other tests, version `version` is the empty string, and `int("")` fails.

Reply:
Gotcha, yeah, no problem.